spec.md

```diff
@@ -300,23 +319,23 @@ that contains both context and data).
   * MUST be unique within the scope of the producer
 
 ### event-time
-* Type: Timestamp per [RFC 3339](https://tools.ietf.org/html/rfc3339)
+* Type: `Timestamp` per [RFC 3339](https://tools.ietf.org/html/rfc3339)
 * Description: Timestamp of when the event happened.
 * Constraints:
   * OPTIONAL
   * If present, MUST adhere to the format specified in
     [RFC 3339](https://tools.ietf.org/html/rfc3339)
 
 ### schema-url
-* Type: URI per [RFC 3986](https://tools.ietf.org/html/rfc3986)
+* Type: `URI` per [RFC 3986](https://tools.ietf.org/html/rfc3986)
```

Comment:
Redundant RFC reference since definition of `URI` already contains a reference?

Reply:
I didn't want to touch that bit, but ok.